database-plugins/src/main/java/co/cask/hydrator/plugin/ConnectionConfig.java

```diff
@@ -77,4 +93,40 @@ public ConnectionConfig() {
     jdbcPluginType = "jdbc";
     enableAutoCommit = false;
   }
+
+  /**
+   * Parses connection arguments into a {@link Properties}.
+   *
+   * @param connectionArguments See {@link ConnectionConfig#connectionArguments}.
+   * @param user See {@link ConnectionConfig#user}.
+   * @param password See {@link ConnectionConfig#password}.
+   */
+  public static Properties getConnectionArguments(@Nullable String connectionArguments,
+                                                  @Nullable String user, @Nullable String password) {
+    KeyValueListParser kvParser = new KeyValueListParser("\\s*;\\s*", "=");
+
+    Map<String, String> connectionArgumentsMap = new HashMap<>();
```

Comment:
We don't name variable with theirs types connectionArguments is fine.

Reply:
That variable was already taken in this context.